tests/test_annotation_stores.py

```diff
@@ -186,7 +201,280 @@ def _fill_store(
+    assert ann._geometry is None
+    assert len(coords.shape) == 2
+    # Check that coords are the same after creating geometry
+    _ = ann.geometry
+    assert ann._geometry is not None
+    geom_coords = ann.coords
+    assert np.array_equal(geom_coords, coords)
+
+
+def test_annotation_multi_point_wkb_coords() -> None:
+    """Test generating coordinates from WKB for MultiPoint.
+
+    This should produce coordinates without generating a Shapely
+    geometry.
+    """
+    multi_point = MultiPoint(
+        [
+            Point(0, 1),
+            Point(2, 3),
+        ],
+    )
+    ann = Annotation(wkb=multi_point.wkb)
+    coords = ann.coords
+    assert len(coords) == len(multi_point.geoms)
+    assert isinstance(coords[0], np.ndarray)
+    assert ann._geometry is None
+    # Check that coords are the same after creating geometry
+    _ = ann.geometry
+    assert ann._geometry is not None
+    geom_coords = ann.coords
+    assert np.array_equal(geom_coords, coords)
+
+
+def test_annotation_multi_polygon_wkb_coords() -> None:
+    """Test generating coordinates from WKB for MultiPolygon.
+
+    This should produce coordinates without generating a Shapely
+    geometry.
+    """
+    multi_poly = MultiPolygon(
+        polygons=[
+            Polygon([[0, 0], [1, 1], [2, 0]]),
+            Polygon([[0, 0], [1, 1], [2, 2], [3, 3]]),
+        ],
+    )
+    ann = Annotation(wkb=multi_poly.wkb)
+    coords = ann.coords
+    assert len(coords) == len(multi_poly.geoms)
+    assert isinstance(coords[0], list)
+    assert ann._geometry is None
+    # Check that coords are the same after creating geometry
+    _ = ann.geometry
+    assert ann._geometry is not None
+    geom_coords = ann.coords
+    assert all(np.array_equal(a, b) for a, b in zip_longest(geom_coords, coords))
+
+
+def test_annotation_multi_line_string_wkb_coords() -> None:
+    """Test generating coordinates from WKB for MultiLineString.
+
+    This should produce coordinates without generating a Shapely
+    geometry.
+    """
+    multi_line_string = MultiLineString(
+        [
+            LineString([(0, 0), (1, 1)]),
+            LineString([(2, 2), (3, 3)]),
+        ],
+    )
+    ann = Annotation(wkb=multi_line_string.wkb)
+    coords = ann.coords
+    assert len(coords) == len(multi_line_string.geoms)
+    assert isinstance(coords[0], np.ndarray)
+    assert ann._geometry is None
+    # Check that coords are the same after creating geometry
+    _ = ann.geometry
+    assert ann._geometry is not None
+    geom_coords = ann.coords
+    assert all(np.array_equal(a, b) for a, b in zip_longest(geom_coords, coords))
+
+
+def test_annotation_coords_wkb_uknown_geometry() -> None:
+    """Test generating coordinates from WKB for unknown geometry type."""
+    ann = Annotation(wkb=b"\x00\x07\x00\x00\x00")
+    with pytest.raises(ValueError, match="Unknown geometry type"):
+        _ = ann.coords
+
+
+def test_annotation_coords_geometry_uknown_geometry(
```

Comment:
typo: uknown again. Also, monkeypatch could be removed as the test doesn't use it?